### PR TITLE
docs: add contribution guide and refactor docs

### LINKED ISSUE #114: Add Contributing Guidelines
I am aware that we have the dev docs, which are great, but we should probably also add a `CONTRIBUTING.md` file for people who are new to tracking bugs and stuff like that. 

Here is [a very simple one](https://github.com/thoughtbot/factory_bot_rails/blob/master/CONTRIBUTING.md) that I like. In general, every template that is older than 5 years should be good as a guideline. Most newer templates are as long as they are woke, and I feel like we can do without adding additional wokeness to this world.

### PR DESCRIPTION
**Pull request by dnlggr**
Resolves #114.

Adds a contribution guide with concise instructions on how to submit a PR, test a version, submit an issue, etc.

Also restructures the existing docs a bit. I moved the guide on how to install the web UI locally to the developer docs since the preferred way of installing for most people is through the node app stores. Running locally and connecting to a remote JoinMarket instance is something for more advanced users I would say.

📚 Preview: [here](https://github.com/joinmarket-webui/joinmarket-webui/tree/contributing-guide)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,95 +35,46 @@ This project builds upon work done by [Shobhitaa](https://github.com/shobhitaa),
...
-To run the web UI locally you need to connect it to a running JoinMarket instance.
-
-#### 1. JoininBox
-
-Install [JoininBox](https://github.com/openoms/joininbox) on your [RaspiBlitz](https://github.com/rootzoll/raspiblitz):
+The alpha version of the JoinMarket Web UI is [available as CLI install in RaspiBlitz v1.7.2](https://github.com/rootzoll/raspiblitz/pull/2747). To install it, exit the Raspiblitz menu and run:
 
 ```
-Services > j [BTC JoinMarket+JoininBox menu]
+config.scripts/bonus.joinmarket-webui.sh on
```
Comment (dnlggr) [inline README.md:43]: Due to #121, we'll probably have to change this to:

```suggestion
config.scripts/bonus.joinmarket-webui.sh on
```

followed by:

```
```
config.scripts/bonus.joinmarket-webui.sh update
```
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
+# Contributing to This Project
+
+This is a [free](https://www.gnu.org/licenses/license-list.html#Expat) and [open-source](https://opensource.org/licenses/MIT) software project and we love receiving pull-requests, bug reports, ideas, and feedback from everyone.
+The aim of this document is to help you get setup for participating in all areas of this project quickly—whether that's submitting code via a pull request, testing the latest iteration, reporting issues, writing documentation, etc.
+
+## Understanding the Components Involved
+
+This project is a web UI for JoinMarket.
```
Comment (MaxHillebrand) [inline CONTRIBUTING.md:8]:
```suggestion
This project is a web UI for [JoinMarket](https://github.com/JoinMarket-Org/joinmarket-clientserver/).
```

**Review by MaxHillebrand — APPROVED** [review]
it's very thorough - great!

**Review by theborakompanioni — APPROVED** [review]
Perfect. You really put a lot of love into these changes. :orange_heart:

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+The web UI's purpose is to be a rather lightweight front end for JoinMarket's API.
```
Comment (dennisreimann) [inline CONTRIBUTING.md:9]:
```suggestion
The web UI's purpose is to be a rather lightweight front end for the JoinMarket API.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+You don't need to worry about that, though.
+To ease development and testing, we provide a Docker setup that runs JoinMarket and its API service in a regtest setup.
+It isn't needed to dig deeper into how it works to use it.
+However, if you want to find out more about it, see [docker/regtest/readme.md](docker/regtest/readme.md).
+
+## Submitting a Pull Request
+
+[Fork](https://github.com/joinmarket-webui/joinmarket-webui/fork), then clone this repo:
+
+```
```
Comment (dennisreimann) [inline CONTRIBUTING.md:21]: Nit for this and other code fencces:

```suggestion
```sh
```

**Review by dennisreimann — APPROVED** [review]
Thanks, very good work!

**Comment by dnlggr** [comment]
Thanks for the feedback guys!

What do you think, should we remove the install instructions for the Raspiblitz until the install is fixed so that it actually works?

**Comment by dennisreimann** [comment]
The fix should be available with the next bugfix release, so I wouldn't mind and keep it in,

**Pull request merged**